extra-tools/index.md

```diff
@@ -28,15 +29,17 @@ extras(ctx, 'markdown', 'display', 'shortcutIcon');
 
 ## Markdown (`markdown`)
 
-You may have noticed most descriptions from annotations are parsed as Markdown. This is thanks to the Markdown filter from [sassdoc-extras][repo_extras] (using [Marked](https://github.com/chjj/marked)).
+You may have noticed most descriptions from annotations are parsed as Markdown. This is thanks to the Markdown filter from [sassdoc-extras][repo_extras] (using [Marked](https://github.com/chjj/marked)). This filter overrides the provided values.
 
-This filter overrides the provided values, so a value such as ``This is some `code`.`` would become ``This is some <code>code</code>``.
+For example, ``This is some `code`.`` becomes ``This is some <code>code</code>``.
 
 ## Display toggle (`display`)
 
-Depending on whether or not you want to display items based on their visibility (access level, `public` or `private`), the `display` filter might be able to help you. When used, it removes items that should not be displayed.
+The `display` filter allows you to display items based on their visibility (access level: `public` or `private`). This filter removes items that should not be displayed.
 
```

Comment:
We should update this part, actually nothing is removed, just a new key added, so it's up to the theme to choose what to display. Not part of this PR anyway.